enterprise/server/raft/cache/BUILD

```
@@ -41,6 +41,9 @@ go_library(
 go_test(
     name = "cache_test",
     srcs = ["cache_test.go"],
+    exec_properties = {
+        "EstimatedComputeUnits": "8",
```

can use 'test.' prefix so that it only applies to the test action and not the build action

```suggestion
        "test.EstimatedComputeUnits": "8",
```